packages/components/src/search/autocomplete.js

```diff
@@ -156,7 +157,7 @@ export class Autocomplete extends Component {
 		const promise = ( this.activePromise = Promise.resolve(
 			typeof options === 'function' ? options( query ) : options
 		).then( optionsData => {
-			if ( ! optionsData ) {
+			if ( ! optionsData || ! this.state.isFocused ) {
```

Comment:
Could we use `state.query`, here? I think it would work as well and this way we wouldn't have to add a new property to the state.
```suggestion
			if ( ! optionsData || ! this.state.query ) {
```

Reply:
Great idea- thanks for the suggestion @Aljullu!  Added in https://github.com/woocommerce/wc-admin/pull/1582/commits/b1a772d782241f21bd606e45dfd061003c66bbd4 and it seems to work just as well.